tests/tests.json

```diff
@@ -255,36 +255,44 @@
 	"params": {"70": 0.0},
 	"parallelization": ["mpi"]
     },
-    "drift-dirichlet-zaxis": {
+    "driftbcs-dirichlet-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "d_inhomo",
 	"rea": "d_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-neumann-zaxis": {
+    "driftbcs-neumann-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "n_inhomo",
 	"rea": "n_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-robin-zaxis": {
+    "driftbcs-robin-zaxis": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "r_inhomo",
 	"rea": "r_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
     },
-    "drift-dirichlet-bdf1r": {
+    "driftbcs-dirichlet-bdf1r": {
 	"app": "drift",
 	"dir": "driftbcs",
 	"usr": "d_inhomo",
 	"rea": "d_inhomo",
 	"params": {"17": -11},
 	"parallelization": ["mpi"]
+    },
+    "newton-dirichlet": {
```

Comment:
Created issue https://github.com/NekCEM/NekCEM/issues/168 to make sure we don't forget.